resources/views/layouts/default.blade.php

```diff
@@ -585,12 +585,11 @@
       <footer class="main-footer">
         <div class="pull-right hidden-xs">
           <b>Version</b> {{ config('version.app_version') }}  build {{ config('version.build_version') }} ({{ config('version.hash_version') }})
-          <a target="_blank" class="btn btn-default btn-xs" href="https://snipe-it.readme.io">Documentation</a>
-          <a target="_blank" class="btn btn-default btn-xs" href="https://snipe-it-manual.readme.io">User's Manual</a>
-          <a target="_blank" class="btn btn-default btn-xs" href="https://snipeitapp.com/support/">Report a Bug</a>
+          <a target="_blank" class="btn btn-default btn-xs hidden" href="https://snipe-it.readme.io">Documentation</a>
```

Comment:
Why? If they aren't to appear on the page remove them completely rather than hiding them?